pkg/cmd/auth/refresh/refresh.go

```diff
@@ -62,16 +65,24 @@ func NewCmdRefresh(f *cmdutil.Factory, runF func(*RefreshOptions) error) *cobra.
 			your gh credentials to have. If no scopes are provided, the command
 			maintains previously added scopes.
 
-			The command can only add additional scopes, but not remove previously
-			added ones. To reset scopes to the default minimum set of scopes, you
-			will need to create new credentials using the auth login command.
+			The --remove-scopes flag accepts a comma separated list of scopes you
+			want to remove from your gh credentials.
+
+			The --reset-scopes flag resets the scopes for your gh credentials to
```

Comment:
```suggestion
			The --reset-scopes flag resets the scopes for your gh credentials to
			the default set of scopes for your auth flow.
```

The default minimum scopes are `repo`, `read:org`, `gist` and in the case of `https` authentication, also `workflow`. I have an open question with older members of the team why the last one for only `https` and not `ssh`. I don't think we need to repeat this information here though.

@shion1305 It looks like the `login` help text is out of date about `gist` since https://github.com/cli/cli/pull/981/files#diff-461484acb1388585e44b6262db2a3e594ad1960aea734741edf4deb2621fab8aR44, perhaps you could open a PR to update that?

Reply:
Actually it's kind of confusing. When we `login` via  we always add `gist` but the minimum scopes required for a token that is [provided are `repo` and `read:org`](https://github.com/cli/cli/blob/baba8949973244a53f6a172a03afa84a7a0ccfc4/pkg/cmd/auth/shared/oauth_scopes.go#L77).